server/src/models/note.model.ts

```
@@ -58,6 +86,19 @@ export class Note {
     return false;
   }
 
+  static async updateNote(
+    noteId: string,
+    filedToEdit: EditableNoteField,
+  ): Promise<boolean> {
+    const note = await firestore.doc(geNotePath(noteId)).get();
+    if (note.exists) {
+      await note.ref.update({ ...filedToEdit });
+      return true;
+    } else {
+      return false;
```

Isn't better to throw an error when the note does not exist?